.dont-break.json

```diff
@@ -0,0 +1,6 @@
+[
+    "https://github.com/ni-kismet/flot-cursors-plugin",
+    "https://github.com/ni-kismet/flot-intensitygraph-plugin",
+    "https://github.com/ni-kismet/flot-charting",
+    "https://github.com/ni-kismet/webcharts"
```

Comment:
Eventually all dependencies will be listed here. At this point there is a bug in dont-break which prevents running tests from packages like webcharts-legends which don't depend directly on engineering-flot.